docs/docs/install/docker-compose.mdx

```diff
@@ -54,22 +52,28 @@ Optionally, you can enable hardware acceleration for machine learning and transc
   <CodeBlock language="bash">{ExampleEnv}</CodeBlock>
 </details>
 
-- Populate custom database information if necessary.
 - Populate `UPLOAD_LOCATION` with your preferred location for storing backup assets.
 - Consider changing `DB_PASSWORD` to a custom value. Postgres is not publically exposed, so this password is only used for local authentication.
   To avoid issues with Docker parsing this value, it is best to use only the characters `A-Za-z0-9`.
 - Set your timezone by uncommenting the `TZ=` line.
+- Populate custom database information if necessary.
+
+:::info Optional Features
+You can make the required edits to `docker-compose.yml` to add external libraries or enable hardware acceleration now by following [their guides](#setting-up-optional-features) up to and including those edits.
```

Comment:
```suggestion
You can edit `docker-compose.yml` to add external libraries or enable hardware acceleration now by following [their guides](#setting-up-optional-features).
```

Reply:
I think this is a fair simplification; it’s just annoying that the optional guides don’t split apart the yaml file edits from the settings changes made to the running program, so users who follow this tip would have to flip-flop between the two pages.